app/ts/utils/constants.ts

```diff
@@ -48,3 +48,23 @@ export const YES_NO_OPTIONS = [
 	'Yes',
 	'No'
 ] as const
+
+export const InvalidRules = [
+  'The market question, resolution details or its outcomes are ambiguous, subjective or unknown.',
+  'The result of the event was known at market creation time.',
+  'The outcome was not known at event expiration time.',
```

Comment:
Is it possible to put the market question creation/resolution dates into these strings directly so users don't have to go look them up?

Reply:
added issue